.mega-linter.yml

```diff
@@ -0,0 +1,2 @@
+ENABLE:
+  - EDITORCONFIG
```

Comment:
I've a question regarding this: Based on the [documentation](https://megalinter.io/latest/descriptors/editorconfig_editorconfig_checker/), you have to set `EDITORCONFIG_EDITORCONFIG_CHECKER ` here, or I'm in the documentation? 🤔

Reply:
Well, we've seen that it's doing its job, haven't we? Not sure what this should change.